src/frontend/metrics/positionToCountries.ts

```diff
@@ -0,0 +1,29 @@
+import borders from '@osm_borders/maritime_10000m';
```

Comment:
This data is accurate within 10 kilometers. There are [other choices with higher resolution][0] but (1) this is small and therefore the best for performance[^1] (2) high accuracy might violate user privacy.

[0]: https://github.com/itisem/osm_borders?tab=readme-ov-file#maritime-borders

[^1]: I assume